src/main/java/hu/oe/nik/szfmv/automatedcar/systemcomponents/Sensor.java

```diff
@@ -25,18 +26,17 @@
     private static final Logger LOGGER = LogManager.getLogger(Main.class);
     private static final int RADAR_RANGE_IN_METER = 3;
     private static final double RADAR_SENSOR_ANGLE = Math.PI / 180 * 50;// 50°
-    private static final Color RADAR_SENSOR_BG_COLOUR = new Color(255, 0, 0, 125);
+    //  private static final Color RADAR_SENSOR_BG_COLOUR = new Color(255, 0, 0, 125);
     private static final int TRIANGLE_POLYGON_POINTS = 3;
     private static final int RADAR_SENSOR_RANGE = RADAR_RANGE_IN_METER * VisualizationConfig.METER_IN_PIXELS;
     // radar sensor triangle data
     private static final int RADAR_TRIANGLE_HALF_X = (int) (Math.tan(RADAR_SENSOR_ANGLE) * RADAR_SENSOR_RANGE);
```

Comment:
A `Sensor`-hoz kellene egy irány ([1,1], [-1,1], ...), amellyel meg tudjuk adni, hogy merre néz ("lokálisan" transzformálja `corner1` és `corner2` értékeket)
ezek is változnak majd az autó forgatásával (TeamA2)